app/views/volunteers/_notes.html.erb

```diff
@@ -22,6 +22,7 @@
                 <td><%= note.created_at.strftime(t("time.formats.standard")) %></td>
                 <td>
                   <%= link_to "Edit", edit_volunteer_note_path(@volunteer, note), class: "btn btn-primary" %>
+                  <%= link_to "Delete", volunteer_note_path(@volunteer, note), class: "btn btn-primary", method: :delete %>
```

Comment:
wondering if would be good to have confirmation here 🤔

Reply:
good point... or maybe just "soft" delete i.e. mark it deleted but don't actually delete it all the way...?